node/overseer/src/lib.rs

```diff
@@ -1767,14 +1729,19 @@ where
 fn spawn<S: SpawnNamed, M: Send + 'static>(
 	spawner: &mut S,
 	futures: &mut FuturesUnordered<BoxFuture<'static, SubsystemResult<()>>>,
-	streams: &mut StreamUnordered<mpsc::Receiver<MaybeTimed<ToOverseer>>>,
+	to_overseer: mpsc::UnboundedSender<MaybeTimed<ToOverseer>>,
```

Comment:
Should we potentially keep the separate channels per subsystem to overseer in order to eventually being able to measure channel fill states? Just an idea.

Reply:
@drahnr I think unless prometheus shows that this channel is really full, we don't need to do that. It'd be a simple change later on.

Reply:
Fair point.